packages/strapi-design-system/src/NumberInput/__tests__/NumberInput.e2e.js

```diff
@@ -45,6 +45,7 @@ test.describe.parallel('NumberInput', () => {
       });
 
       test('puts the step value in the input when pressing ArrowUp and that the input is empty', async ({ page }) => {
+        await page.fill('input', '');
```

Comment:
@ronronscelestes

Also its a bit strange that this methods was removed from tests.
Default iframe for this tests is `iframe.html?id=design-system-components-numberinput--base&viewMode=story`
[Link to vercel](https://design-system-git-main-strapijs.vercel.app/iframe.html?id=design-system-components-numberinput--base&viewMode=story)

Don't know how is even possible to [navigate before each test to this iframe](https://github.com/strapi/design-system/blob/main/packages/strapi-design-system/src/NumberInput/__tests__/NumberInput.e2e.js#L10) with default value of `3.14159265359` and after get `1` after increment. I think it should be fixed this way. Also we can use other iframe (any other empty) for this tests.

Actually i would like to suggest you creating new 'describe' with goto to [this one](https://design-system-git-main-strapijs.vercel.app/iframe.html?id=design-system-components-numberinput--with-initial-empty&viewMode=story).

If you are OK with that just tell me and i`ll commit changes.